background/logging.js

```diff
@@ -0,0 +1,32 @@
+require('app-module-path').addPath(__dirname + '../../app');
+
+var path = require('path');
+
+var nconf = require('nconf');
+var bunyan = require('bunyan');
+
+formatBunyanInfoLog = function() {};
+formatBunyanInfoLog.prototype.write = function (rec) {
+    console.log('[%s] %s: %s',
+        rec.time,
+        bunyan.nameFromLevel[rec.level],
+        rec.msg);
+};
+
+logger = bunyan.createLogger({
+    name: 'Bitcannon',
```

Comment:
This should also be showing that the logs are coming from the web interface since we wanted to modalise this and break it into separate processes.